components/busola-migrator/internal/busola/init.go

```diff
@@ -0,0 +1,63 @@
+package busola
+
+import (
+	"encoding/base64"
+	"fmt"
+	"net/url"
+
+	"github.com/pkg/errors"
+	"k8s.io/client-go/rest"
+
+	"github.com/kyma-project/kyma/components/busola-migrator/internal/config"
+)
+
+const initTemplate string = `{
+  "cluster": {
+    "server": "%s",
+    "certificate-authority-data": "%s"
+  },
+  "auth": {
+    "issuerUrl": "%s",
+    "clientId": "%s",
+    "scope": "%s",
+    "usePKCE": %t
+  },
+  "config": {
+    "disabledNavigationNodes": "",
+    "systemNamespaces": "istio-system knative-eventing knative-serving kube-public kube-system kyma-backup kyma-installer kyma-integration kyma-system natss kube-node-lease kubernetes-dashboard serverless-system"
+  },
+  "features": {
+    "bebEnabled": false
+  }
+}`
+
+func BuildInitURL(appCfg config.Config, kubeConfig *rest.Config) (string, error) {
+	if kubeConfig == nil {
```

Comment:
Is it necessary? 
If kubeConfig was empty line 22 from file `main.go` call `os.Exit(1)`

Reply:
I think it's worth checking if the variable is `nil` even if it's verified in some other place, before the function call, because there is always the possibility that someone changes the code in `main.go` and this might result in unexpected panic.